website/www/site/content/en/documentation/io/built-in/google-bigquery.md

```diff
@@ -667,38 +667,30 @@ You can set `with_auto_sharding=True` to enable dynamic sharding (starting
 The sharding behavior depends on the runners.
 {{< /paragraph >}}
 
-BigQueryIO uses streaming inserts in the following situations:
+BigQueryIO uses Write API in the following situations:
 
 {{< paragraph class="language-java" wrap="span" >}}
 * When you apply a BigQueryIO write transform to an unbounded `PCollection`.
-* When you specify streaming inserts as the insertion method using
-  `BigQueryIO.write().withMethod(STREAMING_INSERTS)`.
+* When you specify Write API as the ingestion method using
```

Comment:
shall we still keep the docs for Streaming_Inserts given it is still used a lot? we can mention `STORAGE_API_AT_LEAST_ONCE` is recommended for sure.

Reply:
cc @ahmedabu98 @Abacn